packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -470,7 +470,9 @@ export default class RestApi {
     api.get(
       '/:collection(vdis|vdi-snapshots)/:object.:format(vhd|raw)',
       wrap(async (req, res) => {
-        const stream = await req.xapiObject.$exportContent({ format: req.params.format })
+        const preferNbd = req.query.preferNbd === 'true'
+        const nbdConcurrency = req.query.nbdConcurrency && parseInt(req.query.nbdConcurrency)
+        const stream = await req.xapiObject.$exportContent({ format: req.params.format, preferNbd, nbdConcurrency })
```

Comment:
Please add documentation for these new params in:

- https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/docs/rest-api.md#vdi-export
- https://github.com/vatesfr/xen-orchestra/blob/master/docs/restapi.md#vm-and-vdi-export